package.json

```diff
@@ -104,7 +104,8 @@
     "uint8array-extras": "^0.5.0",
     "utm": "^1.1.1",
     "validate-color": "^2.2.4",
-    "zustand": "^4.4.6"
+    "zustand": "^4.4.6",
+    "expo-build-properties": "~0.11.1"
```

Comment:
nit: move this so that it's alphabetically sorted (an unfortunate behavior of `npx expo install ...`)

Reply:
https://github.com/digidem/comapeo-mobile/pull/448/commits/4fc064286e0af9563e98d2a4adbd6271f7c4c7c0